lib/utils.js

```diff
@@ -111,6 +112,7 @@ exports.fuse = function fuse (target, handlers) {
 // merges a object's proppertys / values with a other object
 exports.merge = function merge (target, obj) {
   for (var i in obj) {
```

Comment:
Might just make more sense to use `Object.keys` here instead so it solve the problem that hasOwnProperty would resolve and introduce an performance boost.